README.md

```diff
@@ -5,16 +5,16 @@ Welcome to Platform! This all-in-one application allows users to view their favo
 
 ## Developers
 ### Front-end:
-- Matthew Shindel  matthew@shindel.com
-- Dana Zack danaezack@gmail.com
-- Edward Gavin Garcia Edwardgavingarcia@yahoo.com
+- Dana Zack: [GitHub](https://github.com/dana-zack), [LinkedIn](https://www.linkedin.com/in/danazack/), [Email](danaezack@gmail.com)
+- Edward Gavin Garcia: [GitHub](https://github.com/EGavinG), [LinkedIn](https://www.linkedin.com/in/egavingarcia/), [Email](Edwardgavingarcia@yahoo.com)
```

Comment:
Great use of linkage to LinkedIn! I forgot my first names 'Edward' for a moment lol